src/components/SlotCalendarEvent/SlotCalendarEvent.tsx

```diff
@@ -19,16 +20,29 @@ export const SlotCalendarEvent: ComponentType<SlotCalendarEventProps> = ({ event
     const popupRef = useRef<HTMLDivElement>(null);
 
     const content = (
-        <div>
-            <Text size="xs" className={s.CreatorName}>
-                {creator?.name}
-            </Text>
-
-            <Text color="textSecondary" size="xs">
-                {title}
-            </Text>
+        <Text size="xs">
+            <Text>{creator?.name}</Text>
+
+            <Text>{title}</Text>
+
             <CalendarEventLinkedSection interviewSection={interviewSection} sectionTitleOnly />
-        </div>
+
+            {nullable(event.unavailableDueToWeekLimit, () => (
+                <Alert
```

Comment:
Also this two optional alert components may be placing outside of the text component

Reply:
Fixed